sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/UseReturnStatement.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
```

Comment:
Something went wrong here as well. There are many files in `sonaranalyzer-dotnet` directory duplicated. They are also in `analyzers` where they should be.